modules/standard/discord/discord_controller.py

```diff
@@ -80,7 +80,7 @@ def start(self):
         self.relay_hub_service.register_relay(self.RELAY_HUB_SOURCE, self.handle_incoming_relay_message)
         self.register_discord_command_handler(self.help_discord_cmd, "help", [])
 
-        self.db.exec("CREATE TABLE IF NOT EXISTS discord (channel_id VARCHAR(64) NOT NULL, server_name VARCHAR(256) NOT NULL, channel_name VARCHAR(256) NOT NULL, relay_ao SMALLINT NOT NULL DEFAULT 0, relay_dc SMALLINT NOT NULL DEFAULT 0)")
+        self.db.exec("CREATE TABLE IF NOT EXISTS discord (channel_id INTEGER(64) NOT NULL UNIQUE, server_name VARCHAR(256) NOT NULL, channel_name VARCHAR(256) NOT NULL, relay_ao SMALLINT NOT NULL DEFAULT 0, relay_dc SMALLINT NOT NULL DEFAULT 0)")
```

Comment:
perhaps this should be a BIGINT instead of INTEGER?  the 64 is not the size of the field like it is with CHAR, it's just a display hint.  see this explanation: https://stackoverflow.com/a/3135854

Reply:
Ahh... that makes sense.. ofc, changing it to BIGINT, after reading the explanation...